presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1528,6 +1532,14 @@ public SystemSessionProperties(
                         "When the size difference between current table and history table exceed this threshold, do not match history statistics",
                         0.0,
                         true),
+                stringProperty(
+                        HISTORY_OPTIMIZATION_PLAN_CANONICALIZE_STRATEGY,
```

Comment:
should we just call this history_optimization_strategy? the current name is a bit long and has many internal details (that we canonicalize plans) + incorrect grammar

Reply:
rename it to `history_based_optimization_plan_canonicalization_strategy`. I think it's OK to have details on the name, after all people who use it should know what they are doing.